src/test/java/com/ge/predix/metering/filter/NuregoIT.java

```diff
@@ -175,23 +178,22 @@ private String retrieveComponent(String serviceInstanceGuid, String nuregoAccess
 			} catch (Exception ex) {
 				LOGGER.error("Cannot retrieve component ID", ex);
 			}
-			Thread.sleep(2000); // need to sleep because nurego mapping takes time to update
+			Thread.sleep(20000); // need to sleep because nurego mapping takes time to update
 		}
 		if(responseEntity == null) {
 			return null;
 		}
 		String responseBody = responseEntity.getBody();
-
 		@SuppressWarnings("unchecked")
 		Map<String,Object> map = new ObjectMapper().readValue(responseBody, Map.class);
 		return map.get(Constants.ID).toString();
 
 	}
 
 
-	private Date tomorrow() {
+	private Date endTime() {
 		final Calendar cal = Calendar.getInstance();
-		cal.add(Calendar.DATE, +1);
+		cal.add(Calendar.DATE, +2);
```

Comment:
We should investigate either changing this to specify UTC time zone (since Nurego retrieve raw usage API returns UTC data) or check if that endpoint has a query parameter to specify time zone so our data is consistent

Reply:
fixed.